files/install-deps-worker.yaml

```diff
@@ -12,7 +12,7 @@
           - nss_wrapper # openshift anyuid passwd madness
           - redis # redis-cli for debugging
           - origin-clients # for sandcastle
-          - python3-kubernetes # for sandcastle
+          # we can't install kube from Fedora because API changes b/w major versions
```

Comment:
Can you keep the `- python3-kubernetes # for sandcastle` there, just comment it out?
Otherwise, it looks like (to me) that this ("we can't install kube...") comment is meant for the below `python3-fedora`.

Also, we don't install kube from Fedora because we need a pinned version which might and might not be in fedora.
(The reason why we use pinned version in sandcastle is "because API changes b/w major versions")

Reply:
good point, I just wanted to be sure mainly it builds fine with the initial commit, the comment should be more clear now